stable/rethinkdb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: rethinkdb
 description: The open-source database for the realtime web
-version: 0.1.3
+version: 0.2.0
```

Comment:
From the looks of this patch, this is a bug fix, right? So it should just bump the patch version?

Reply:
If I was updating the documentation I would agree with you, but since I'm updating the implementation to match the docs this change is breaking and thus it should be a major version update.

Reply:
*minor, not major. But ya, I agree.